src/style.css

```diff
@@ -46,7 +46,14 @@
 }
 
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-family:
+    "Aileron",
+    Inter,
+    system-ui,
+    Avenir,
+    Helvetica,
+    Arial,
+    sans-serif;
```

Comment:
Since we ship "Aileron" ourselves, we may remove some of the fallback options.

Reply:
Yes, we can likely go with a slightly shorter list, probably `"Aileron", system-ui, sans-serif` or the like.